packages/harmony/src/components/button/PlainButton/PlainButton.mdx

```diff
@@ -1,22 +1,100 @@
-import { Meta, Title, Description, Primary, Stories } from '@storybook/blocks'
+import { Meta, Title, Description, Controls, Canvas, Primary, Stories, Unstyled } from '@storybook/blocks'
 
-import { Heading, BetaWarning } from 'storybook/components'
+import { IconCaretDown, IconCloudUpload } from 'icons'
+import { ComponentRules, Heading, BetaWarning } from 'storybook/components'
 
+import { Box } from '../../layout'
+import { PlainButtonSize } from '../types'
+import { PlainButton } from './PlainButton'
 import * as PlainButtonStories from './PlainButton.stories'
 
 <Meta of={PlainButtonStories} />
 
-<Title>Plain Button</Title>
+<Title />
 
 <BetaWarning />
 
 - [Overview](#overview)
 - [Props](#props)
+- [Variants](#variants)
+- [Configuration options](#configuration-options)
+- [Use cases and examples](#use-cases-and-examples)
+- [Do's and dont's](#dos-and-donts)
+- [Related Components](#related-components)
 
 <Heading>
   ## Overview
   <Description />
 </Heading>
-<Primary />
 
-<Stories />
+<Canvas of={PlainButtonStories.Overview} />
+
+## Props
+
+<Controls />
+
+## Variants
+
+### Default
+
+<Canvas of={PlainButtonStories.Default} />
+
+### Subdued
+
+<Canvas of={PlainButtonStories.Subdued} />
+
+### Inverted
+
+<Canvas of={PlainButtonStories.Inverted} />
+
+## Configuration Options
+
+### Sizes
+
+<Canvas of={PlainButtonStories.Sizes} />
+
+### Leading Icons
+
+<Canvas of={PlainButtonStories.LeadingIcons} />
+
+### Trailing Icons
+
+<Canvas of={PlainButtonStories.TrailingIcons} />
+
+## Use cases and examples
+
+- Plain buttons should be used as a tertiary button.
+- Plain buttons are flexible and adaptable to any design. They can be emphasized or hidden.
+- Plain iconas are different from text link in there they may contain a leading or trailing icon.
+
+## Do's and don'ts
+
+<ComponentRules
+  rules={[
+    {
+      positive: {
+        description: 'Use plain buttons to reveal or hide additional information.',
+        size: 'small',
+        component: (
+          <Unstyled> 
+            <PlainButton size={PlainButtonSize.LARGE} iconRight={IconCaretDown}>
+              See More
+            </PlainButton>
+          </Unstyled>
+        )
+      },
+      negative: {
+        description: 'Don’t frame a plain button to create a “mock” secondary button.',
+        size: 'small',
+        component: (
+          <Unstyled> 
+            <Box pv='m' ph='xl' border='strong' borderRadius='s'>
+              <PlainButton size={PlainButtonSize.LARGE} iconLeft={IconCloudUpload}>Upload</PlainButton>
+            </Box>
+          </Unstyled>
+        )
+      }
+    }
+  ]}
+/>
+
```

Comment:
related components?

Reply:
Oh right, there weren't any in the figma docs. Button and SocialButton sound good here?